### ISSUE TITLE
Shebang syntax highlighting breaks on empty newline

### ISSUE THREAD

**Issue by chanderson0** [nefrob/vscode-just#63]
It seems like blank lines in shebang recipes cause subsequent lines to stop being syntax highlighted as that shell. To be fair, it looks like the Github Markdown highlighting also has this issue.

```just
# Works
recipe1:
  #!/usr/bin/env bash
  echo "Hello, World!"
  var=1

# Works
recipe2:
  #!/usr/bin/env bash
  echo "Hello, World!"
  
  # Note two spaces in above line
  var=2
  echo $var

# Doesn't work
recipe3:
  #!/usr/bin/env bash
  echo "Hello, World!"

  var=2
  echo $var
```

<img width="401" alt="Screenshot 2024-12-12 at 2 05 05 PM" src="https://github.com/user-attachments/assets/3761a88a-81e3-41ff-a370-98a48ab57d77" />

**Comment by nefrob** [comment]
Yeah I think this is a limitation of textmate grammars. We can apply a simple rule to treat all indented text after the recipe definition as the recipe body, but can't parse the document in a more complex way to track the body after no indentation.

We could try and be clever to check if after no indentation are lines are empty then the next indentation could be treated as body again. I'll play around with this basic case but likely won't be able to expand beyond that.

On the github preview, that is because this repo is used for github syntax highlighting.

**Comment by nefrob** [comment]
Resolved in #69 and will go out with the next release.
